main.c

```diff
@@ -80,7 +80,7 @@ main(int argc, char *argv[])
 		}
 
 	argc -= optind;
-	argv += optind;
+	
```

Comment:
I would leave this in place because keeping argc and argv in sync is a very common idiom.
If this code is ever modified later to use argv again then this missing argv update could easily lead to confusion.

Reply:
I have re-added argv per Stefans advice thanks for the review

Reply:
thanks!